### DISCUSSION TITLE
[version 8.2.7] Class name must be a valid object or a string

### DISCUSSION THREAD

**Discussion by carlomigueldy** [spatie/laravel-medialibrary#1906] [Q&A] [answered]
Hi,

While I followed the steps for installing the package, I am receiving this error. Or was there something that I missed?

![image](https://user-images.githubusercontent.com/45052332/83342232-6e6a5f00-a31f-11ea-98d4-004090bac06c.png)

**Comment by jallowmamadou** [comment, marked as answer]
I had the same issue. all I did was to completely remove the package including the service provider and config. i event delete it from the vendor directory and then followed this thread: https://github.com/spatie/laravel-medialibrary/issues/1567#issuecomment-632375721
and reinstalled all over again and everything works find for me now: https://docs.spatie.be/laravel-medialibrary/v8/installation-setup/

**Reply by Shohan494** [reply]
That helped @jallowmamadou 
Cheers
